JSON-connect/.gitignore

```diff
@@ -1,3 +0,0 @@
-node_modules
-package-lock.json
-.clasp.json
```

Comment:
And these?

Reply:
All of these are old and no-longer relevant.